Client/Frontend/Browser/BrowserViewController/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -81,6 +96,13 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
 
         actions.append(getLibraryActions(vcDelegate: self))
         actions.append(getOtherPanelActions(vcDelegate: self))
+
+        if let whatsNewAction = whatsNewAction, var lastGroup = actions.last, lastGroup.count > 1 {
+            lastGroup.insert(whatsNewAction, at: lastGroup.count - 1)
```

Comment:
Isn't this the same as lastGroup.append(whatsNewAction)

Reply:
oops! yes

Reply:
nvm, that is correct, it inserts replacing that position, so that represents the second last position. `append` is equivalent to 'insert(..., at: count)'